File: packetCapture/src/main/java/com/b_knabe/packet_capture/core/util/common/IOUtils.java
package com.b_knabe.packet_capture.core.util.common;

import java.io.Closeable;
import java.io.IOException;

public class IOUtils {
    /**
     * Try to close all resources
     *  Input and Output utility for files
     * @param resources IO streams, etc...
     */
    public static void close(Closeable... resources) {
        for (Closeable resource : resources) {
            if (resource == null) continue;
            try {
                resource.close();
            } catch (IOException ignored) {
            }
        }
    }
}

File: packetCapture/src/main/java/com/ninpou/packetcapture/core/util/common/IOUtils.kt
package com.ninpou.packetcapture.core.util.common

import java.io.Closeable
import java.io.IOException

object IOUtils {
    /**
     * Try to close all resources
     *  Input and Output utility for files
     * @param resources IO streams, etc...
     */
    fun close(vararg resources: Closeable?) {
        for (resource in resources) {
            if (resource == null) continue
            try {
                resource.close()
            } catch (ignored: IOException) {
            }
        }
    }
}